cmd/crio/main.go

```diff
@@ -343,6 +346,10 @@ func main() {
 			Value: 9090,
 			Usage: "port for the metrics endpoint",
 		},
+		cli.BoolFlag{
+			Name:  "read-only",
+			Usage: "setup all non privileged containers to run as read-only",
```

Comment:
Sould "non privileged" be "unprivileged"?

Reply:
yup, fixed.